### ISSUE TITLE
CSV download includes Laravel debugbar scripts and styles

### ISSUE THREAD

**Issue by ankurk91** [rap2hpoutre/fast-excel#49]
Hi,

* Laravel - 5.6.37
* php 7.1
* fast-excel 0.7.0
* [Laravel debugbar](https://github.com/barryvdh/laravel-debugbar) 3.1.5

When we have laravel debugbar installed and try to download csv, the csv file includes the laravel debugbar scripts and styles along with actual records.

```php
// Code to reproduce the issue
Route::get('test', function () {
    return (new \Rap2hpoutre\FastExcel\FastExcel(\App\Models\User::all()))->download('users-list.csv');
});
```

```php
// How i fixed this issue
Route::get('test', function () {
    return response()->streamDownload(function () {
        return (new \Rap2hpoutre\FastExcel\FastExcel(\App\Models\User::all()))->download('users-list.csv');
    });
});
```
If possible please add this piece of code to docs somewhere.

Thanks.

**Comment by rap2hpoutre** [comment]
Hi, thank your for your contribution! And thank you for the workaround you provided! 👍 

> If possible please add this piece of code to docs somewhere.

I think it could be even better to actually add your code in the lib itself, in order to make it work for everybody without having to read the doc! I just pushed this here: https://github.com/rap2hpoutre/fast-excel/commit/c87837f7ec8d9743bdb2055518755a3411830446.

I will release it ASAP!

**Comment by rap2hpoutre** [comment]
🎉 Available in v0.8.0 thanks again!
